### ISSUE TITLE
Can't find the properties file

### ISSUE THREAD

**Issue by White7561** [StevenMassaro/SonarrCalendarSync#1]
I tried using it. I really need it but everytime I tried to run it. Itd spit error saying it can't detect the properties file. I'm using a Mac. Ofc I already made the file. But anyway. Just to make sure. Could you send the file insides with fake info as real info? To make sure the format is right since idk if you need say the API inside " < > " or not. Anyway i tried java 8 and latest 10. Thanks eh

**Comment by wikkixwikki** [comment]
I am also having the same issue where the prefs.properties file is "No such file or directory" when it is right there in the same directory.

It is showing 
```
java.io.FileNotFoundException: /home/devin/sonarrcal\prefs.properties (No such file or directory)
        at java.io.FileInputStream.open0(Native Method)
        at java.io.FileInputStream.open(FileInputStream.java:195)
        at java.io.FileInputStream.<init>(FileInputStream.java:138)
        at SonarrCalendarSync.Base.<init>(Base.java:39)
        at SonarrCalendarSync.Runner.<init>(Runner.java:17)
        at SonarrCalendarSync.SonarrCalendarSync.main(SonarrCalendarSync.java:13)
Exception in thread "main" java.lang.NullPointerException
        at SonarrCalendarSync.Base.<init>(Base.java:45)
        at SonarrCalendarSync.Runner.<init>(Runner.java:17)
        at SonarrCalendarSync.SonarrCalendarSync.main(SonarrCalendarSync.java:13)
```

could the issue be the slash (\) instead of needing to be / where it is looking? see the path after sonarrcal

**Comment by StevenMassaro** [comment]
[SonarrCalendarSync-1.1-jar-with-dependencies.zip](https://github.com/StevenMassaro/SonarrCalendarSync/files/2968413/SonarrCalendarSync-1.1-jar-with-dependencies.zip)

I think you're right, on Unix systems, the slash was the wrong direction. Try this and let me know if it works.

**Comment by StevenMassaro** [comment]
Resolved in release 1.1.
